src/main/webapp/style.css

```diff
@@ -1,5 +1,11 @@
 #content {
-    margin-left: auto;
-    margin-right: auto;
-    width: 650px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+}
+
+#welcome {
+  font-size: 18;
+  text-align: center;
+  margin-bottom: 10px;
```

Comment:
Not a fan of specifying font sizes myself. l prefer using h1, h2, etc. to denote different font sizes. lt's more elegant that way and easier to track font sizes. l think we should try to Bootstrap as much as we can in general.
https://getbootstrap.com/docs/4.0/content/typography/

Reply:
Done